### PR TITLE
Don't override bg color of gtk-themes

### PR DESCRIPTION
**Pull request by raveit65**
A lot of themes use theme_bg_color as define for their css.
So, don't override their settings with fedora's blue color
fixes https://bugzilla.redhat.com/show_bug.cgi?id=1684808
fixes https://github.com/rhinstaller/anaconda/issues/1859

### CODE REVIEW

File: data/anaconda-gtk.css [hunk 1]
```diff
@@ -84,7 +84,7 @@ levelbar.discrete trough block.filled.high {
 
 /* theme colors/images */
 
-@define-color theme_bg_color @fedora;
+@define-color fedora_bg_color @fedora;
```
Comment (poncovka) [inline data/anaconda-gtk.css]: The color can be set to a different value in different products. Maybe it could be called `product_bg_color` then?

**Comment by poncovka** [comment]
Please, rebase it to `f30-devel`. Then the fix will get to Fedora 30 and Rawhide.

**Comment by raveit65** [comment]
ok, i will change it to `product_bg_color` and rebase against f30-devel

**Comment by jkonecny12** [comment]
Please fix your source branch. After the rebase you have here 2 commits not related to this feature.

**Comment by raveit65** [comment]
@jkonecny12 
You mean i should remove the complete commit body?

**Comment by jkonecny12** [comment]
@raveit65 No, I mean remove the commit mess which have got here by changing the target branch in this PR.
Please look on [commits tab](https://github.com/rhinstaller/anaconda/pull/1860/commits) above in this PR.

**Comment by raveit65** [comment]
The commit mass was trigered by a `git rebase f30-devel` on master, i didn't create an extra branch before. :/
Let me try to fix it.

**Comment by jkonecny12** [comment]
Don't worry, it happened a few times to me too :)

**Comment by raveit65** [comment]
Closing this , i will make a new fresh PR, this is easier as reverting a rebase on master

**Comment by raveit65** [comment]
New PR https://github.com/rhinstaller/anaconda/pull/1862

**Pull request closed without merging**